src/web/core/src/main/resources/GeoServerApplication.properties

```diff
@@ -979,6 +979,10 @@ resetFilter = Reset
 
 save = Save
 
+back = Back
+
+notice = Make sure to save the layer configuration to get the SQLView changes applied
```

Comment:
The translation key `notice` is a bit general, usually we should have a naming convention like `back.description` (if that is what this is used for).

Reply:
You could make the argument for an `apply` button here, which would apply the entire layer editor changes (including this sub page). This would be handy as it would also the view to be revised quickly for debugging.